screens/Map/Map.tsx

```diff
@@ -65,22 +74,20 @@ export const Map = (props: MapProps) => {
           latitudeDelta: 0.009,
           longitudeDelta: 0.009,
         }}>
-        {MOCKDATA.map((data) => {
-          if (data.type === filter || filter === 'all')
-            return (
-              <Marker
-                onPress={() => {
-                  navigation.navigate('Review', { id: data.lazy });
-                }}
-                image={getMarker(data.type)}
-                key={data.id}
-                coordinate={{
-                  latitude: data.latitude,
-                  longitude: data.longtitude,
-                }}
-              />
-            );
-        })}
+        {reviewData.map((review) =>  (
+            <Marker
+              onPress={() => {
+                navigation.navigate('Review', { id: review.kratooID });
+              }}
+              image={getMarker(review.type)}
+              key={review.kratooID}
+              coordinate={{
+                latitude: review.postions.lat,
+                longitude: review.postions.lng,
+              }}
+            />
+          );
```

Comment:
It shouldn't have ; (It gives an error when build the app, so remove ; please)

```suggestion
          )
```